Polyhedron/demo/Polyhedron/Plugins/Operations_on_polyhedra/Surface_mesh_to_polyhedron_plugin.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Scene_polyhedron_item.h"
+#include "Scene_surface_mesh_item.h"
+
+#include <CGAL/Three/Polyhedron_demo_plugin_interface.h>
+#include <CGAL/boost/graph/copy_face_graph.h>
+#include <QString>
+#include <QAction>
+#include <QMenu>
+#include <QMainWindow>
+#include <QApplication>
+#include <QTime>
+#include <QMessageBox>
+using namespace CGAL::Three;
+class Surface_mesh_to_polyhedron_plugin :
+  public QObject,
+    public Polyhedron_demo_plugin_interface
+{
+  Q_OBJECT
+  Q_INTERFACES(CGAL::Three::Polyhedron_demo_plugin_interface)
+  Q_PLUGIN_METADATA(IID "com.geometryfactory.PolyhedronDemo.PluginInterface/1.0")
+
+public:
+
+void init(QMainWindow* mw,
+          Scene_interface* scene_interface,
+          Messages_interface* )
+{
+    scene = scene_interface;
+    this->mw = mw;
+    QAction *actionToPoly = new QAction("Convert to Polyhedron", mw);
+    QAction *actionToSM = new QAction("Convert to Surface_mesh", mw);
+    actionToPoly->setProperty("subMenuName",
+                              "FaceGraph Conversion");
+    actionToSM->setProperty("subMenuName",
+                             "FaceGraph Conversion");
+
```

Comment:
Trying to follow the naming convention, these 2 actions should be inside the menu "BGL" (and not "FaceGraph Conversion"), because BGL is the package they belong to.
For now we did not find any strategy that would be more consistent than this one.